### PR TITLE
Investigate xmltestfailures

### PR DESCRIPTION
**Pull request by bladehr8**

### CODE REVIEW

**Comment by msotclas** [comment]
Hi __@bladehr8__, I'm your friendly neighborhood Microsoft Open Technologies, Inc. Pull Request Bot (You can call me MSOTBOT). Thanks for your contribution!
    <span>
        In order for us to evaluate and accept your PR, we ask that you sign a contribution license agreement. It's all electronic and will take just minutes. I promise there's no faxing. https://cla2.msopentech.com.
    </span>

TTYL, MSOTBOT;

File: test/Microsoft.AspNet.Mvc.FunctionalTests/InputObjectValidationTests.cs [hunk 1]
```diff
@@ -43,7 +43,7 @@ public static IEnumerable<object[]> SimpleTypePropertiesModelRequestData
             }
         }
 
-        [Fact]
```
Comment (Eilon) [inline test/Microsoft.AspNet.Mvc.FunctionalTests/InputObjectValidationTests.cs]: Was this attribute removed on purpose?

File: test/Microsoft.AspNet.Mvc.FunctionalTests/InputObjectValidationTests.cs [hunk 2]
```diff
@@ -43,7 +43,7 @@ public static IEnumerable<object[]> SimpleTypePropertiesModelRequestData
...
+#if !ASPNETCORE50
         public async Task CheckIfObjectIsDeserializedWithoutErrors()
```
Comment (Eilon) [inline test/Microsoft.AspNet.Mvc.FunctionalTests/InputObjectValidationTests.cs]: So this test is the only one that fails on ASPNETCORE50 now?

**Comment by Eilon** [comment]
BTW all the ASPNET/ASPNETCORE stuff was renamed to DNX so you'll want to update that in the PR.

**Comment by rynowak** [comment]
this has been fixed now.

**Pull request closed without merging**